dpypx/autodraw.py

```diff
@@ -58,7 +55,7 @@ def load(cls, client: Client, data: str) -> AutoDrawer:
         for _ in range(height):
             row = []
             for _ in range(width):
-                row.append(Pixel.from_hex(lines.pop(0)))
+                row.append((Pixel.from_hex(lines.pop(0))), True)
```

Comment:
```suggestion
                row.append((Pixel.from_hex(lines.pop(0)), True))
```
I think the brackets are slightly misplaced here.